README.md

```diff
@@ -1,63 +1,57 @@
-<h3 align=center><a href="#build-framework"><img src=".github/armbian-logo.png" alt="Armbian logo" width="144"></a><br>
-build framework</h3>
-
-<p align=right>&nbsp;</p>
+<p align="center">
+  <a href="#build-framework">
+  <img src=".github/armbian-logo.png" alt="Armbian logo" width="144">
+  </a><br>
+  <strong>armbian build framework</strong><br>
+</p>
 
 [![GitHub last commit (branch)](https://img.shields.io/github/last-commit/armbian/build/master)](https://github.com/armbian/build/commits)
 [![GitHub Workflow Status](https://img.shields.io/github/workflow/status/armbian/build/Build?label=build%20train)](https://github.com/armbian/build/actions/workflows/build-train.yml)
 [![Twitter Follow](https://img.shields.io/twitter/follow/armbian?style=flat-square)](https://twitter.com/intent/follow?screen_name=armbian)
+[![Join the Discord server](https://img.shields.io/discord/854735915313659944.svg?color=7289da&label=Discord%20server&logo=discord)](https://discord.gg/gNJ2fPZKvc)
```

Comment:
Perhaps just "Discord" ? without "server"

Reply:
that's depend on you :)

Reply:
This line of badges is already wide and a word "server" doesn't really add value